inference-engine/samples/CMakeLists.txt

```diff
@@ -145,6 +145,9 @@ endif()
 # exactly the same OpenCV_DIR path which was used for the InferenceEngine build
 if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/common/format_reader")
     add_subdirectory(common/format_reader)
+elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/common")
+    # opencv_wrapper_c for C samples is here
+    add_subdirectory(common)
```

Comment:
it is better to isolate opencv_wraper like done for cpp samples, for example, common/opencv_c_wraper

Reply:
I've included your gflags commit  + changed the subproject name in #2278 to fix this issue.